data/vendors/octolis.json

```diff
@@ -0,0 +1,20 @@
+{
+    "slug": "octolis",
+    "name": "octolis",
+    "logoUrl": "/vendors/octolis.png",
```

Comment:
This is not linked to any logo in the repository, please submit your logo at the same time as the JSON =)

Reply:
![octolis](https://user-images.githubusercontent.com/100789766/227493091-49e5e233-304a-4a6e-9da6-0090d295f7db.png)